p/properties/properties_ubi_8.4.sh

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : properties
+# Version       : v1.8.5
+# Source repo   : https://github.com/magiconair/properties
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Gururaj R Katti <Gururaj.Katti@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=main
```

Comment:
```suggestion
  export VERSION=v1.8.5
```
Let's default to version for which script is created instead of main branch. Rest all looks good. Thanks

Reply:
ok, @gururajrkatti I think the confusion here is regarding using "main" and I think you have used "master" in some other places. This is not wrong per se, however we would like to standardize this at some point so that we build specific versions as master/main are active branches that constantly move which would imply that my definition of "latest version" would mean the latest released version or tag and I think @Siddhesh-Ghadi would be okay if you are able to default to that. Please correct me @Siddhesh-Ghadi if that's not what you had in mind here